### ISSUE TITLE
Feature request, set bed during day

### ISSUE THREAD

**Issue by Ganjalf-Amsterdoom** [ForgeEssentials/ForgeEssentials#2089]
Is it possible to add setting /bed by right click the bed during the day? If the percentual of sleeping people is reached to make the day pass happens that some people cannot set their bed position and have to ask other to not sleep.
Thank you.

**Comment by spacebuilder2020** [comment]
I will look into it.  It is defiantly something useful, but we also are going to need to work on getting 1.8.9 stable, and start working on 1.9.4

**Comment by Ganjalf-Amsterdoom** [comment]
Thank you very much for looking into it.

**Comment by spacebuilder2020** [comment]
np

**Comment by Zethalion** [comment]
As we moved to 1.12.2, is this still an issue? Closing this issue in 7 days if no response @Ganjalf-Amsterdoom

**Comment by spacebuilder2020** [comment]
This is a feature request that is relevant to 1.12 so it will remain open as a new feature for 1.12.

**Comment by Ganjalf-Amsterdoom** [comment]
Yes please, would be awesome if it could be implemented for 1.12

**Comment by spacebuilder2020** [comment]
In 1.12, there is a forge hook for bed events so we should be able to hook into it.  The only caveat is that since the /bed command uses the player's spawnPoint internally, to set the /bed command, we will either have to rewrite the command to use an alternate point, or just set the spawnPoint in a hook.
